lib/gem_of.rb

```diff
@@ -0,0 +1,108 @@
+require "rubygems" # for Gem Versioning methods
+
+# Namespace for Gem methods
+module GemOf
+  # produce a list of gems for use in a gems Gemfile
+  # @return [String] a string of Gemfile dependencies for a Gemfile to eval
+  # @example eval this in your Gemfile in its binding
+  #   eval(GemOf.gems, binding)
+  # @api public
+  class Gems
+    def initialize
+      set_gem_versions
+
+      @gem_code = <<-HEREDOC
+      source "https://rubygems.org"
+      # place all development, system_test, etc dependencies here
+
+      # lint/unit tests
+      gem "rake"
+      gem "rototiller", "~> 1.0"
+      gem "rspec",      "~> 3.4.0"
+      gem "rubocop",    "~> 0.49.1" # used in tests. pinned
+      gem "simplecov",  "~> 0.14.0" # used in tests
+      gem "yardstick",  "~> 0.9.0"  # used in tests
+      gem "markdown",   "~> 0"
+      gem "flay",       "~> 2.10.0" # used in tests
+      gem "flog",       "~> 4.6.0"  # used in tests
+      gem "roodi",      "~> 5.0.0"  # used in tests
+      gem "rubycritic"
+      gem "coveralls",  require: false # used in tests
+
+      group :system_tests do
+        gem "beaker",        GemOf.location_of(ENV["BEAKER_VERSION"] ||
+          "#{@beaker_version}")
+        gem "beaker-hostgenerator"
+        gem "beaker-abs",    GemOf.location_for(ENV["BEAKER_ABS_VERSION"] ||
+          "~> 0.2")
+        gem "nokogiri"       ,"#{@nokogiri_version}"
+        gem "public_suffix"  ,"#{@public_suffix_version}"
+        #gem "activesupport" ,"#{@activesupport_version}"
+      end
+
+      local_gemfile = "Gemfile.local"
+      if File.exists? local_gemfile
+        eval(File.read(local_gemfile), binding)
+      end
+
+      user_gemfile = File.join(Dir.home,".Gemfile")
+      if File.exists? user_gemfile
+        eval(File.read(user_gemfile), binding)
+      end
```

Comment:
Did you test this with local or user gemfiles? Won't this duplicate it since the bundle install should also check for those by default?

Reply:
i have a ~/.Gemfile with pry in it, for instance.  use it all the time.  
what do you mean 'duplicate it'?
if the user has the same gems in their gemfile, it could create a problem, yeah

Reply:
I just mean that the code in the local and user gemfiles should already be executed, not sure why we need to also call them explicitly here?

Reply:
i don't think bundler looks for local nor user "other" gemfiles.   have you tried it without these lines?

Reply:
http://bundler.io/man/gemfile.5.html#NAME